test/known_issues/test-stdout-buffer-flush-on-exit.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/2148
+
+require('../common');
+const assert = require('assert');
+const execSync = require('child_process').execSync;
+
+const longLine = 'foo bar baz quux quuz aaa bbb ccc'.repeat(65536);
+
+if (process.argv[2] === 'child') {
+  process.on('exit', () => {
+    console.log(longLine);
+  });
+  process.exit();
+}
+
+const stdout = execSync(`${process.execPath} ${__filename} child`)
+  .toString()
```

Comment:
nit: maybe these would look a little less oddly placed if they were lined up with the first `e` in `execSync()`?